Platform/iOS/Legacy/ja.lproj/Main.strings

```diff
@@ -0,0 +1,531 @@
+"Ijr-H4-Hee.text" = "デフォルト";
+
+/* Class = "UILabel"; text = "Isolate Guest"; ObjectID = "iS1-Ur-HFa"; */
+"iS1-Ur-HFa.text" = "ゲストをホストから隔離する";
+
+/* Class = "UILabel"; text = "Select..."; ObjectID = "iuP-Eh-P68"; */
+"iuP-Eh-P68.text" = "選択します...";
+
+/* Class = "UILabel"; text = "Host Address (IPv6)"; ObjectID = "j8H-o8-7gQ"; */
+"j8H-o8-7gQ.text" = "ホストアドレス (IPv6)";
+
+/* Class = "UILabel"; text = "DNS Search Domains"; ObjectID = "jDl-dl-Cu8"; */
+"jDl-dl-Cu8.text" = "DNS検索ドメイン";
+
+/* Class = "UITextField"; placeholder = "Default"; ObjectID = "Jk5-aT-3ap"; */
+"Jk5-aT-3ap.placeholder" = "デフォルト";
+
+/* Class = "UILabel"; text = "ac97"; ObjectID = "Jp2-wl-rfZ"; */
+"Jp2-wl-rfZ.text" = "ac97";
+
+/* Class = "UILabel"; text = "Sharing"; ObjectID = "jv0-6r-NAJ"; */
+"jv0-6r-NAJ.text" = "共有";
+
+/* Class = "UILabel"; text = "Removable"; ObjectID = "K1N-PB-wfm"; */
+"K1N-PB-wfm.text" = "リムーバブル";
+
+/* Class = "UILabel"; text = "Default"; ObjectID = "KFq-K6-6KS"; */
+"KFq-K6-6KS.text" = "デフォルト";
+
+/* Class = "UILabel"; text = "Network Card"; ObjectID = "KRc-3d-cbz"; */
+"KRc-3d-cbz.text" = "ネットワークカード";
+
+/* Class = "UILabel"; text = "Font Size"; ObjectID = "KSo-te-FQs"; */
+"KSo-te-FQs.text" = "フォントサイズ";
+
+/* Class = "UITableViewSection"; headerTitle = "Name"; ObjectID = "kxc-0J-xgO"; */
+"kxc-0J-xgO.headerTitle" = "名前";
+
+/* Class = "UILabel"; text = "Detail"; ObjectID = "KYQ-B4-Cf3"; */
+"KYQ-B4-Cf3.text" = "詳細";
+
+/* Class = "UILabel"; text = "Default"; ObjectID = "l4L-0j-IXJ"; */
+"l4L-0j-IXJ.text" = "デフォルト";
+
+/* Class = "UILabel"; text = "JIT Cache"; ObjectID = "led-8Q-nNh"; */
+"led-8Q-nNh.text" = "JITキャッシュ";
+
+/* Class = "UITableViewSection"; headerTitle = "Mouse Wheel"; ObjectID = "lFm-mY-36G"; */
+"lFm-mY-36G.headerTitle" = "マウスホイール";
+
+/* Class = "UILabel"; text = "Enabled"; ObjectID = "Llu-cx-Osx"; */
+"Llu-cx-Osx.text" = "有効";
+
+/* Class = "UITableViewSection"; footerTitle = "Command to send when resizing the console. Placeholder $COLS is the number of columns and $ROWS is the number of rows."; ObjectID = "LTq-Mi-VuJ"; */
+"LTq-Mi-VuJ.footerTitle" = "コンソールのサイズを変更するときに送信するコマンド。プレースホルダー $COLS は列の数、 $ROWS  は行の数に置き換えられます。";
+
+/* Class = "UITableViewSection"; headerTitle = "Resize Console Command"; ObjectID = "LTq-Mi-VuJ"; */
+"LTq-Mi-VuJ.headerTitle" = "Resize Console Command";
+
+/* Class = "UILabel"; text = "Title"; ObjectID = "lws-wx-k0R"; */
+"lws-wx-k0R.text" = "Title";
+
+/* Class = "UILabel"; text = "MB"; ObjectID = "MAs-NR-G0n"; */
+"MAs-NR-G0n.text" = "MB";
+
+/* Class = "UITableViewSection"; footerTitle = "Default JIT Cache size is 1/4 of the allocated Memory size. This is in addition to the host memory! Force Multicore will allow multi-CPU emulation even if the memory model is not supported by QEMU (x86 on ARM for example). This could result in incorrect emulation."; ObjectID = "MGc-rW-PLa"; */
+"MGc-rW-PLa.footerTitle" = "デフォルトの0に設定すると、割り当てられたメモリサイズの1/4が使われます。これはホストメモリに含まれません!";
```

Comment:
This lacks a translation for "Force Multicore will allow multi-CPU emulation even if the memory model is not supported by QEMU (x86 on ARM for example). This could result in incorrect emulation."